src/renderer/viz/expressions/belongs.js

```diff
@@ -91,8 +91,14 @@ function generateBelongsExpression (name, inlineMaker, jsEval) {
 
         _bindMetadata (meta) {
             super._bindMetadata(meta);
-            checkType(name, 'input', 0, 'category', this.input);
-            checkType(name, 'list', 1, 'category-list', this.list);
+            const validTypes = ['number', 'category', 'date'];
```

Comment:
so are we finally supporting also 'dates' here?

Reply:
Sure, good catch, I'm going to remove it for now 👍

Reply:
I think I was reviewing a previous commit, sorry. 😄